config/locales/javascript/javascript.en.yml

```diff
@@ -239,9 +239,9 @@ en:
       contacts: "Contacts"
       posts: "Posts"
 
-    conversation:
-      new:
-        no_contacts: "You need to add some contacts before you can start a conversation."
```

Comment:
Hmm... Indeed. I must have failed somewhere. It didn't work the first time. Ok, I correct that and push.